Polyhedron/demo/Polyhedron/Scene_item.h

```diff
@@ -37,6 +30,14 @@ class SCENE_ITEM_EXPORT Scene_item : public QObject {
   Q_ENUMS(RenderingMode)
   Q_PROPERTY(RenderingMode renderingMode READ renderingMode WRITE setRenderingMode)
 public:
+  enum OpenGL_program_IDs { PROGRAM_WITH_LIGHT,
+                            PROGRAM_WITHOUT_LIGHT,
+                            PROGRAM_WITH_TEXTURE,
+                            PROGRAM_WITH_TEXTURED_EDGES,
+                            PROGRAM_INSTANCED,
+                            PROGRAM_INSTANCED_WIRE,
+                            NB_OF_PROGRAMS };
+
```

Comment:
@maxGimeno: The use of macros was rather ugly. Here the enum definition defines the same constants. As a bonus, you also get the constant `NB_OF_PROGRAMS` for free (here its value is 5).